cassandra_repair_scheduler.py

```diff
@@ -493,13 +502,13 @@ def cli_parsing():
     parser.add_argument("--reset", action="store_true", default=False,
                         help="Reset the repair status for the host")
     options = parser.parse_args()
+    options.ttl_repair = args.ttl_repair if args.ttl_repair else args.ttl
```

Comment:
where does args.ttl_repair come from?  Shouldn't that be options.ttl_repair?  Likewise args.ttl?